srcpkgs/cyme/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'cyme'
+pkgname=cyme
+version=2.0.0
+revision=1
+build_style=cargo
```

Comment:
```suggestion
build_style=cargo
checkdepends="eudev"
```

The failing test is missing the database provided by this

Reply:
That would explain why I can't reproduce locally! (and maybe why aarch64 mysteriously passed, but none of the others did - something different about those runners?)

Thanks a ton, I'm glad my hunch about this being a GitHub Actions issue was wrong.

Reply:
tests are only run on host archs, not on cross such as aarch64